pipeline_dp/dp_engine.py

```diff
@@ -533,7 +534,67 @@ def _drop_partitions_under_threshold(self, col):
                                     lambda row: row[1].privacy_id_count != None,
                                     "Drop partitions under threshold")
 
-    def _annotate(self, col, params: pipeline_dp.SelectPartitionsParams,
+    def anonymize_values(
+        self,
+        col,
+        params: pipeline_dp.aggregate_params.AnonymizeValuesParams,
+        out_explain_computation_report: Optional[
+            pipeline_dp.ExplainComputationReport] = None):
+        """Anonymizes values by adding noise.
+
+        It anonymizes values per partition key, when it is known the
```

Comment:
Maybe mention the rationale for this method which is, IIUC, ability to apply DP to pre-aggregated data when user identifiers are not available. 

This could be something like:
 "This method allows applying differential privacy to pre-aggregated data. This relies on the assumption that the sensitivities of the pre-aggregated values are known, and the partitiion keys are public or generated with DPEngine.select_partitions".

Reply:
Thanks